.jvmopts

```diff
@@ -1,6 +1,5 @@
 -Xss4m
--Xms1G
--Xmx2G
+-Xmx1G
```

Comment:
Do we need to change `.jvmopts`? Did you find that these were not neccessary? I think I changed these for the CI :thinking:

Reply:
Ach, sorry, I was running `.bin/test.sh` which changes these automatically. I will revert this.

Reply:
Ach, so `-Xmx1G` is actually used for the CI, totally forgot